sway-core/src/asm_lang/allocated_ops.rs

```diff
@@ -486,6 +488,7 @@ impl fmt::Display for AllocatedInstruction {
             JMPF(a, b) => write!(fmtr, "jmpf {a} {b}"),
             JNZB(a, b, c) => write!(fmtr, "jnzb {a} {b} {c}"),
             JNZF(a, b, c) => write!(fmtr, "jnzf {a} {b} {c}"),
+            JAL(a, b, c) => write!(fmtr, "jal  {a} {b} {c}"),
```

Comment:
Nit two spaces before `{a}`.

Reply:
This is on purpose. The output format is neatly aligned with all opcodes taking 4 characters. You can see other branches here already doing that.